pyanaconda/installation.py

```diff
@@ -318,7 +319,8 @@ def run_pre_install():
 
         if can_install_bootloader:
             payload.requirements.add_packages(storage.bootloader.packages, reason="bootloader")
-        if flags.flags.cmdline.getbool("fips"):
+        if kernel_arguments.get("fips", default=False, interpret_booleans=True) and \
+                "nofips" not in kernel_arguments :
```

Comment:
I couldn't found any proof any the `nofips` option is documented or used. It is always documented as `fips=1`.

Reply:
Removed.